Procfile

```diff
@@ -1,2 +1,4 @@
 web: vendor/bin/heroku-php-apache2 /public/
 
+release: php artisan migrate --force
```

Comment:
```
Some migration operations are destructive, which means they may cause you to lose data. In order to protect you from running these commands against your production database, you will be prompted for confirmation before the commands are executed. To force the commands to run without a prompt, use the --force flag:
```
Does this mean that it always prompts you for any migrations in production or that it knows which ones in particular are destructive?

Reply:
Yeah, if `APP_ENV` is `production` it'll always ask for user confirmation.

Reply:
(Which is considerate if you're SSHing in and don't realize you're not on your local dev, but not particularly helpful when trying to do something in a script!)